src/app/modules/create-user/services/create-user.service.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { CreateUser } from 'src/app/common/interfaces/createUser';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CreateUserService {
+
+  constructor(private http: HttpClient) { }
+
+  private url = 'localhost:8080';
```

Comment:
si ce faci daca se schimba url din localhost in altceva? stai sa modifici variabila asta in toate serviciile? :))